src/publish.proj

```diff
@@ -1,41 +1,52 @@
-    <ItemGroup>
-      <ForPublishing Include="$(PublishPattern)" />
-    </ItemGroup>
-    <!-- add relative blob path metadata -->
-    <ItemGroup>
-      <ForPublishing>
-        <RelativeBlobPath Condition="'$(PublishTestNativeBins)' != 'true'">$(__BuildType)/%(RecursiveDir)%(Filename)%(Extension)</RelativeBlobPath>
-        <RelativeBlobPath Condition="'$(PublishTestNativeBins)' == 'true'">$(__BuildType)/%(RecursiveDir)%(Filename)%(Extension)</RelativeBlobPath>    
-      </ForPublishing>
-    </ItemGroup>
-    <Error Condition="'@(ForPublishing)' == ''" Text="No items were found matching pattern '$(PublishPattern)'." />
-  </Target>
+  <PropertyGroup>
+    <FeedsDir Condition="'$(BuildOS)' == 'Windows_NT'">$(MSBuildThisFileDirectory)../component/Microsoft.Dotnet.Build.Tasks.Feed/desktop/</FeedsDir>
+    <FeedsDir Condition="'$(BuildOS)' != 'Windows_NT'">$(MSBuildThisFileDirectory)../component/</FeedsDir>
+  </PropertyGroup>
+
+  <Import Project="$(FeedsDir)Publish.targets" />
+
+  <PropertyGroup>
+    <PackagesPattern Condition="'$(PackagesPattern)' == '' and '$(PublishTestNativeBins)' != 'true'">$(PackagesBinDir)pkg\*.nupkg</PackagesPattern>
+    <PackagesPattern Condition="'$(PackagesPattern)' == '' and '$(PublishTestNativeBins)' == 'true'">$(OutputPath)\bin\**</PackagesPattern>
+    <SymbolsPackagesPattern>$(PackagesBinDir)symbolpkg\*.nupkg</SymbolsPackagesPattern>
+  </PropertyGroup>
 
   <PropertyGroup>
-    <PublishPattern Condition="'$(PublishPattern)' == '' and '$(PublishTestNativeBins)' != 'true'">$(PackagesBinDir)**\*.nupkg</PublishPattern>
-    <PublishPattern Condition="'$(PublishPattern)' == '' and '$(PublishTestNativeBins)' == 'true'">$(OutputPath)\bin\**</PublishPattern>
+    <ContainerName Condition="'$(__Container)' == '' and '$(PublishTestNativeBins)' != 'true'">coreclr-$(PreReleaseLabel)-$(BuildNumberMajor)-$(BuildNumberMinor)</ContainerName>
+    <ContainerName Condition="'$(__Container)' == '' and '$(PublishTestNativeBins)' == 'true'">coreclr-$(PreReleaseLabel)-$(BuildNumberMajor)-$(BuildNumberMinor)-$(__DistroRid)-$(__BuildArch)</ContainerName>
+    <ContainerName Condition="'$(__Container)' != '' and '$(PublishTestNativeBins)' != 'true'">$(__Container)</ContainerName>
+    <ContainerName Condition="'$(__Container)' != '' and '$(PublishTestNativeBins)' == 'true'">$(__Container)-$(__DistroRid)-$(__BuildArch)</ContainerName>
+    <ContainerName>$(ContainerName.Replace(".","-"))</ContainerName>
+    <AccountName>$(CloudDropAccountName)</AccountName>
+    <AccountKey>$(CloudDropAccessToken)</AccountKey>
   </PropertyGroup>
 
-  <Target Name="CreateContainerName"
-          DependsOnTargets="CreateVersionFileDuringBuild"
-          Condition="'$(ContainerName)' == '' or '$(PublishTestNativeBins)' == 'true'">
+  <Target Name="PublishPackages" BeforeTargets="PublishOutputLeg" Condition="'$(__PublishPackages)' == 'true'">
+    <PropertyGroup>
+      <FeedType>Blob</FeedType>
+      <RelativePath>packages</RelativePath>
```

Comment:
@karajas @weshaggard I'm thinking this should be changed to match what we have now (Inside of containers, top-level folders are named Release/Debug/Checked, each of which has packages/symbols subdirectories). Is a specific structure needed, or should I switch to use the old layout?